src/rust/en.tsumino/res/source.json

```diff
@@ -0,0 +1,21 @@
+{
+  "info": {
+    "id": "en.tsumino",
+    "lang": "en",
+    "name": "Tsumino",
+    "version": 1,
+    "url": "https://www.tsumino.com/",
+    "nsfw": 2
+  },
+  "listings": [
+    { "name": "Newest" },
+    { "name": "Oldest" },
+    { "name": "Alphabetical" },
+    { "name": "Rating" },
+    { "name": "Pages" },
+    { "name": "Views" },
+    { "name": "Random" },
+    { "name": "Comments" },
+    { "name": "Popularity" }
```

Comment:
no point having sorting methods as listings. probably just have `New` and `Popular` as listings.

Reply:
I'm going to keep the oldest category as well as just the new and popular listings